riot_api_helpers.py

```diff
@@ -42,4 +42,66 @@ def get_summoner_by_name(summoner_name):
             "summoner_level": json_response_content["summonerLevel"]
         }
     except (KeyError, TypeError, ValueError):
-        return None
\ No newline at end of file
+        return None
+
+def get_matches_for_account(account_id, champion=None, queue=None, season=None, endTime=None, beginTime=None, endIndex=None, beginIndex=None):
```

Comment:
Is there a way/would it be better to allow an arbitrary number of kwargs and allow the server to handle them, rather than enforcing the keys internally? This approach makes it more closely tied with Riot's API, so if they change the name of a parameter this is less flexible.